src/main/wrapper/ExecutionService.ts

```diff
@@ -183,6 +184,10 @@ export class ExecutionService {
                   const scans = CxScan.parseProject(resultObject);
                   cxCommandOutput.payload = scans;
                   break;
+                case CxConstants.SCAN_VORPAL:
+                    const vorpal = CxVorpal.parseScan(resultObject);
```

Comment:
If the CLI returns an error (not the object) Its should know how to handle it too and parse to a string then pass tit.